app/client/screens/ForgotPassword.tsx

```
@@ -0,0 +1,94 @@
+  withZodForm<ForgotPasswordFormType>(ForgotPasswordForm);
+
+const ForgotPassword = () => {
+  const [currentStep, setCurrentStep] = useState(0);
+
+  const handleNextStep = () => {
+    if (currentStep < WIZARD_LABELS.forgotPassword.length - 1) {
+      setCurrentStep((prev) => prev + 1);
+    }
+  };
+
+  const handlePreviousStep = () => {
```

ESLint

[error] 'handlePreviousStep' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)